manifests/cli.pp

```diff
@@ -22,9 +22,10 @@
         -> Anchor['jenkins::end']
   }
 
-  $jar = "${jenkins::libdir}/jenkins-cli.jar"
-  $extract_jar = "jar -xf ${jenkins::libdir}/jenkins.war WEB-INF/jenkins-cli.jar"
-  $move_jar = "mv WEB-INF/jenkins-cli.jar ${jar}"
+
+  $jar = "${jenkins::libdir}/cli.jar"
```

Comment:
What's the reason this needs to change? The overall change would be smaller if we didn't and since we control this file, there should be no issue. Am I missing something?

Reply:
do you mean why the name is cli.jar instead of jenkins-cli.jar?

Reply:
Yes

Reply:
> Do you happen to know which version introduced this change? It'd be nice to set a minimum version number in the README.
not sure, the latest i had was 2.210-1.1 and it worked with jenkins-cli.jar

Reply:
> Yes

no reason. Thought the origin file was cli-$version.jar, that's why renamed. Can change it back if you want.

Reply:
I'm always a fan of the smallest possible changes. That makes it easier to review.

Reply:
done